go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/fhitchen/vault-plugin-database-couchbase
+
+go 1.14
+
+require (
+	github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 // indirect
+	github.com/cenkalti/backoff v2.2.1+incompatible // indirect
+	github.com/containerd/continuity v0.0.0-20200413184840-d3ef23f19fbb // indirect
+	github.com/couchbase/gocb/v2 v2.1.1
+	github.com/docker/go-connections v0.4.0 // indirect
+	github.com/docker/go-units v0.4.0 // indirect
+	github.com/hashicorp/errwrap v1.0.0
+	github.com/hashicorp/go-version v1.2.0
+	github.com/hashicorp/vault v1.4.2
+	github.com/hashicorp/vault/api v1.0.5-0.20200317185738-82f498082f02
+	github.com/hashicorp/vault/sdk v0.1.14-0.20200429182704-29fce8f27ce4
+	github.com/mitchellh/mapstructure v1.3.0
+	github.com/opencontainers/image-spec v1.0.1 // indirect
+	github.com/opencontainers/runc v0.1.1 // indirect
+	github.com/ory/dockertest v3.3.5+incompatible
+	github.com/sirupsen/logrus v1.6.0 // indirect
```

Comment:
Updated the main to use hclog and used just t.Log in the tests. Not sure if this will remove it from the go.mod though as I never added it.

Reply:
Normally an unused package will be removed when you run `go mod tidy`, although I just had a closer look at this and it's actually a dependency of the `dockertest` package, so it will still stick around after removing its direct usages. In the PR's current state, there are a few other packages that will be removed by a tidy though.

Reply:
Ran go mod tidy.